### ISSUE TITLE
Improve wording and link-text of text above the Search form

### ISSUE THREAD

**Issue by AlOneill** [FreeUKGen/MyopicVicar#2362]
The text above the search form is:
You may access the original version of Freecen by clicking [this link]

This is not helpful for a screen-reader user (who can call up a list of links on a page) because 'this link' could be anything.
Also, the casing of 'Freecen' should be 'FreeCEN'.

Suggest that the text would be better as:
You can use this link to visit the [original version of FreeCEN]

**Comment by AlOneill** [comment]
Wrong board, so closing.
